Makefile

```diff
@@ -1,28 +1,54 @@
-R_DYNTRACE_DIRECTORY = R-dyntrace
-EVALDYNTRACER_DIRECTORY = evaldyntracer
-VANILLA_R = R
-R_DYNTRACE_PULL_FLAGS =
-EVALDYNTRACER_PULL_FLAGS =
-R_DYNTRACE_BUILD_COMMAND = cd $(R_DYNTRACE_DIRECTORY); ./build
-EVALDYNTRACER_BUILD_COMMAND = $(R_DYNTRACE_DIRECTORY)/bin/R CMD INSTALL --build --with-keep.source $(EVALDYNTRACER_DIRECTORY)
+export R_ENABLE_JIT=0
+export R_DISABLE_BYTECODE=1
+export R_COMPILE_PKGS=0
 
-setup:
-	git clone git://github.com/PRL-PRG/R-dyntrace.git ${R_DYNTRACE_DIRECTORY}
-	$(R_DYNTRACE_BUILD_COMMAND)
-	git clone git://github.com/PRL-PRG/evaldyntracer.git $(EVALDYNTRACER_DIRECTORY)
-	$(EVALDYNTRACER_BUILD_COMMAND)
-	$(VANILLA_R) --file=scripts/install-dependencies.R
+R_DYNTRACE_HOME := R-dyntrace
+R_DYNTRACE := $(R_DYNTRACE_HOME)/bin/R
+OUTPUT_DIRPATH := $(shell date +'%Y-%m-%d-%H-%M-%S')
+ANALYSIS_DIRPATH := $(OUTPUT_DIRPATH)/analysis
+RAW_ANALYSIS_DIRPATH := $(ANALYSIS_DIRPATH)/raw
+CORPUS_DIRPATH := $(OUTPUT_DIRPATH)/corpus
+LOG_DIRPATH := $(OUTPUT_DIRPATH)/log
+JOBLOG_FILEPATH := $(LOG_DIRPATH)/summary.csv
+CORPUS_FILEPATH := data/corpus.csv
```

Comment:
Isn't it just a text file - one package at a line? Thus `corpus.txt` or even just `packages.txt`?